### ISSUE TITLE
Add a NavigationMenu.Sub example

### ISSUE THREAD

**Issue by andrecasal** [radix-ui/primitives#2257]
The NavigationMenu primitive doesn't have an example usage of the `<NavigationMenu.Sub>`. Looking at the [example provided](https://www.radix-ui.com/docs/primitives/components/navigation-menu), it's unclear why it's not using `<NavigationMenu.Sub>` and [what it's for](https://601857eb614bae0021c9b9dd-vibchwvkwb.chromatic.com/?path=/story/components-navigationmenu--submenus) (I've never seen that navigation pattern before, but ok 😅).

### TL;DR
Add an example usage of the `<NavigationMenu.Sub>` to the [official docs](https://www.radix-ui.com/docs/primitives/components/navigation-menu).

**Comment by benoitgrelard** [comment]
There is an example already: https://www.radix-ui.com/primitives/docs/components/navigation-menu#with-submenus
